app/src/main/java/com/elmeradrianv/shesafe/LaunchActivity.java

```diff
@@ -0,0 +1,23 @@
+package com.elmeradrianv.shesafe;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.os.Handler;
+
+public class LaunchActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_launch);
+        getSupportActionBar().hide();
+
+        new Handler().postDelayed(() -> {
```

Comment:
Is this powering a splashscreen? I'd add a comment mentioning that's the intent (ha).